app/component/itinerary/itinerary-walk-distance.cjsx

```diff
@@ -0,0 +1,18 @@
+React = require 'react'
+Icon  = require '../icon/icon'
+moment = require 'moment'
+
+ItineraryWalkDistance = (props) ->
+
+  ItineraryWalkDistance.description =
+    "Displays the total walk distance of the itinerary"
+
+  ItineraryWalkDistance.propTypes =
+    itinerary: React.PropTypes.object.isRequired
+
+  <span>
+    &nbsp;&nbsp;<Icon img={'icon-icon_walk'}/>
```

Comment:
Do we need to add the spaces here, shouldn't that be done with css padding/margin?